docs/deployment-guide.md

```diff
@@ -2,16 +2,149 @@
+
+    ```bash
+    export NEXUS_USERNAME="<your-username>"
+    export NEXUS_PASSWORD="<your-password>"
+    ```
+
+8. Run the main vagrant:
+
+    ```bash
+    vagrant up
+    ```
+
+    > NOTE: The playbook takes around 30 minutes to complete and mostly depends on your internet connection speed.
+
+Once ACS has initialized access the system using the following URLs using a browser:
+
+* Digital Workspace: `http://192.168.56.100/workspace`
+* Share: `http://192.168.56.100/share`
+* Repository: `http://192.168.56.100/alfresco`
+* API Explorer: `http://192.168.56.100/api-explorer`
+
+To access the machine vagrant created and ran the playbook on use `vagrant ssh`.
+
+## Getting started with localhost deployment
+
+If you have access to a pristine host running one of the supported Linux distributions, you can follow this quickstart for a localhost deployment.
+
+### Download playbook
+
+If you're not working directly working on the control node, transfer the ZIP
+file to the control node together with the SSH private key required to login to
+the target machines, and SSH into the machine:
+
+```bash
+scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
+scp  -i ~/.ssh/ansible_rsa user@controlnode:.ssh
+ssh  user@controlnode
+unzip alfresco-ansible-deployment-<version>.zip
+cd alfresco-ansible-deployment
+```
+
+You can also use Git to fetch latest sources (or a specific release for example by adding `-b v2.1.0`) on the control node with:
+
+```bash
+git clone https://github.com/Alfresco/alfresco-ansible-deployment.git
+cd alfresco-ansible-deployment
+```
+
+> You may want to generate an SSH key pair locally and use it later for
+> deployment. Wether you generate one or you use one you copied over to the
+> control node, it is your responsibility to deploy it to the target machines so
+> Ansible can use it. Using SSH keys is recommended but not mandatory. If using
+> password instead make sure to add the `-k`switch to the ansible command so it
+> prompts you for a password.
+
+### Setup runtime environment
+
+Before starting using the playbook, make sure you are running at least python 3.8:
+
+```bash
+python3 --version
+pip install --user pipenv
+```
+
+We made mandatory the usage of [pipenv](https://pipenv.pypa.io/en/latest/) to
+make sure that you will run the playbook with the same set of python
+dependencies we are running our integration tests.
+
+Install pipenv via pip:
+
+```bash
+pip install --user pipenv
+```
+
+If `pip` is not available, you can bootstrap pipenv using the [crude
+installation](https://pipenv.pypa.io/en/latest/install/#crude-installation-of-pipenv):
+
+```bash
+curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python3
+```
+
+Now you are ready to install Ansible and required runtime dependencies in a dedicated
+virtual environment managed by pipenv.
+
+Run from the playbook folder:
+
+```bash
+pipenv install --deploy
```

Comment:
I've removed the flag forcing to use our pinned python version, python 3.8+ should always work and no need to bother the user with that